app/containers/List/ListItem.tsx

```diff
@@ -129,28 +132,35 @@ const Content = React.memo(
 				accessibilityLabel={handleAcessibilityLabel}
 				accessibilityRole={accessibilityRole ?? 'button'}>
 				{left ? <View style={styles.leftContainer}>{left()}</View> : null}
-				<View style={styles.textContainer}>
-					<View style={styles.textAlertContainer}>
-						<Text style={[styles.title, styleTitle, { color: color || themes[theme].fontDefault }]} numberOfLines={1}>
-							{translateTitle && title ? I18n.t(title) : title}
-						</Text>
-						{alert ? (
-							<CustomIcon
-								name='info'
-								size={ICON_SIZE}
-								color={themes[theme].buttonBackgroundDangerDefault}
-								style={styles.alertIcon}
-							/>
+				{title || subtitle ? (
+					<View style={styles.textContainer}>
+						<View style={styles.textAlertContainer}>
+							{typeof title === 'string' ? (
+								<Text style={[styles.title, styleTitle, { color: color || themes[theme].fontDefault }]}>
+									{translateTitle && title ? I18n.t(title) : title}
+								</Text>
+							) : (
+								title()
```

Comment:
`title || subtitle`, so if `!!subtitle` and `!title`, `title()` is going to be called and the app is going to crash.